webapp/src/app/bcviewer/bcviewer.component.ts

```diff
@@ -188,13 +188,27 @@ class LinkBlock {
     index: number;
     height: number;
     maxHeight: number;
+    sign: string;
```

Comment:
It's not definitley assigned in the constructor (ie it can be `undefined`): either change it to `string | undefined` or set it.

Reply:
set to `""` by default.